airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/base/errors/messages/RedshiftErrorMessage.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.base.errors.messages;
+
+import static io.airbyte.integrations.base.errors.utils.ConnectionErrorType.INCORRECT_ACCESS_PERMISSION;
+import static io.airbyte.integrations.base.errors.utils.ConnectionErrorType.INCORRECT_DB_NAME;
+import static io.airbyte.integrations.base.errors.utils.ConnectionErrorType.INCORRECT_HOST_OR_PORT;
+import static io.airbyte.integrations.base.errors.utils.ConnectionErrorType.INCORRECT_USERNAME_OR_PASSWORD;
+import static io.airbyte.integrations.base.errors.utils.ConnectorName.REDSHIFT;
+
+import io.airbyte.integrations.base.errors.utils.ConnectorName;
+
+public class RedshiftErrorMessage extends ErrorMessage {
+
+  {
+    CONSTANTS.put("28000", INCORRECT_USERNAME_OR_PASSWORD);
+    CONSTANTS.put("HY000", INCORRECT_HOST_OR_PORT);
```

Comment:
Based on the [Redshift Error Code](https://docs.aws.amazon.com/redshift/latest/mgmt/rsql-query-tool-error-codes.html) `HY000` error code doesn't seem to exist. Please clarify me if there's another place where this exists? 

Also based on some of the codes I've seen so far, the error messages that have been defined seem to be too general and actually lead users astray because they're not one-to-one match of what the underlying database's error code means. How did these specific error codes get selected?